test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -171,12 +171,39 @@ public class ControllerActionInvokerTest
         }
 
         [Fact]
-        public async Task InvokeAction_InvokesAsyncExceptionFilter_ShortCircuit()
+        public async Task InvokeAction_InvokesExceptionFilter_ShortCircuit_ExceptionHandled()
         {
             // Arrange
             var filter1 = new Mock<IExceptionFilter>(MockBehavior.Strict);
 
+            var filter2 = new Mock<IExceptionFilter>(MockBehavior.Strict);
+            filter2
+                .Setup(f => f.OnException(It.IsAny<ExceptionContext>()))
+                .Callback<ExceptionContext>(context =>
+                {
+                    filter2.ToString();
```

Comment:
These calls to `ToString()` aren't needed. This is an **ancient** workaround for an issue in Moq. Just don't put them in new code